File: option/src/app/node_none/src/rest.js
const express = require('express')
const app = express()
const port = 8080

app.get('/info', (req, res) => {
    res.send('NodeJS - Express / No Database')
})

app.get('/dept', (req, res) => {
    res.send('[ \
        { "deptno": "10", "dname": "ACCOUNTING", "loc": "Seoul"}, \
        { "deptno": "20", "dname": "RESEARCH", "loc": "Cape Town"}, \
        { "deptno": "30", "dname": "SALES", "loc": "Brussels"}, \
        { "deptno": "40", "dname": "OPERATIONS", "loc": "San Francisco"} \
    ]')
})

app.listen(port, () => {
    console.log(`OCI Starter: listening on port ${port}`)
})

File: option/src/app/python_none/src/rest.py
import os 
from app import app
from flask import jsonify
from flask import flash, request

@app.route('/dept')
def dept():
    return jsonify( [ 
        { "deptno": "10", "dname": "ACCOUNTING", "loc": "Seoul"},
        { "deptno": "20", "dname": "RESEARCH", "loc": "Cape Town"},
        { "deptno": "30", "dname": "SALES", "loc": "Brussels"},
        { "deptno": "40", "dname": "OPERATIONS", "loc": "San Francisco"}
    ] )

@app.route('/info')
def info():
        return "Python - Flask / No Database"          

if __name__ == "__main__":
    app.run(host="0.0.0.0", port=8080)